index.html

```diff
@@ -64,9 +73,11 @@ <h1>
 </section>
 <section class="works" >
     <div class="recent-works">
-        <h2>My Recent </br> Works</h2>
+        <h2 class="mobile-work-header">My Recent </br> Works</h2>
+        <h2 class="desktop-work-header">My Recent Works</h2>
     </div>
-    <div class="myworks-card">
```

Comment:
- [ ] The works section from your design is inconsistent with the design template as shown below, there is little or no spacing between each works card, please kindly add spacing using `grid-gap` since they are grid items. :+1: 
- figma design
![sp-f](https://user-images.githubusercontent.com/89055502/236341755-26d73e35-b2a0-4461-8078-140195fd2f6f.png)
- your design
![sp=s](https://user-images.githubusercontent.com/89055502/236341849-bb58e51e-7355-4694-9d7f-ad71bc2e3687.png)